sw/device/silicon_creator/lib/rom_print.c

```diff
@@ -6,10 +6,28 @@
 
 #include <assert.h>
 #include <stdarg.h>
+#include <stdbool.h>
 #include <stdint.h>
 
 #include "sw/device/silicon_creator/lib/drivers/uart.h"
 
+void print_integer(unsigned value, bool is_signed) {
+  char buf[12];
+  char *b = buf + sizeof(buf);
+  if (is_signed && (int)value < 0) {
+    uart_putchar('-');
+    value = (unsigned)(-(int)value);
+  }
+  *--b = '\0';
+  do {
+    *--b = '0' + value % 10;
+    value /= 10;
+  } while (value);
+  while (*b) {
+    uart_putchar(*b++);
+  }
+}
+
 rom_error_t rom_printf(const char *format, ...) {
```

Comment:
We should probably call this module `debug_print.c` and this function `debug_printf()` to avoid enabling this in production builds. Otherwise this would become a target for pentesting.

Reply:
Renamed as `dbg_print` and renamed the printing function as `dbg_printf()`.